README.md

```diff
@@ -1542,6 +1542,238 @@ data: {
 
 ```
 
+## Payment Channels
+
+Nakamoto style transaction replacement channels are supported in TXQ.
+
+https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-April/002417.html
+
+Use payment channels when you wish to coordinate actions amongst 2 or more parties, but do not wish to settle the transaction until
+nLockTime and the final state of the data is acceptable
+
+Note: Payment "Channels" have nothing directly related to the queue channel name used in the TXQ api. The name 'paymentChannelId' is used to distinguish the usage. In context below 'paymentChannelId' is a global identifier (preferred random uuid) that can be used by the client
+to create, update, and manage the Nakamoto replacement payment channel.
+
+### Create Payment Channel
+
+Choose a globally unique identifier for `paymentChannel` and create it. Optionally provide a `rawtx` for the initial transaction statte.
+
+Once a payment channel is initialized with the first transaction, then all subsequent updates must use the same inputs and adhere to the rules
+regarding nSequence, nLockTime. See: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-April/002417.html
+
+`POST /api/v1/paymentchannel/:paymentChannelId`
+
+```javascript
+{
+  "rawtx": '003222...' // Optional initial rawtx to initialize with. If left empty then use "Updates" to change

```

Comment:
Just to confirm, so if I initialize a payment channel with TXQ and the server is restarted before the channel is closed, I would need to re-initialize the payment channel with `rawtx` of the original initiation and this would work. Is that correct?

Reply:
No. The payment channel will be persisted to DB, so if TXQ reboots you can continue where you left off.

I haven't updated the database schema yet, as this is just a proposal first to make sure it's the correct thing to build.